### PR TITLE
chore: Easier to use grpc telemetry 

### PR DESCRIPTION
**Pull request by frondeus**

### CODE REVIEW

File: crates/rpc/src/materializer_general.rs [hunk 1]
```diff
@@ -5,10 +5,21 @@ use tonic::transport::Channel;
...
-    GeneralMaterializerClient::connect(addr)
+    connect_inner(addr)
         .await
         .map_err(|err| ClientError::ConnectionError {
             service: "materializer_general",
             source: err,
         })
 }
+
+async fn connect_inner(
```
Comment (Kononnable) [inline crates/rpc/src/materializer_general.rs:16]: Not sure if its worth to extract such functions just for the purpose of having one-liners.

File: crates/tracing_tools/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+version = "0.1.0"
+edition = "2018"
+license = "Apache-2.0"
+
+[lib]
+name = "tracing_tools"
+path = "src/lib.rs"
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[features]
```
Comment (Kononnable) [inline crates/tracing_tools/Cargo.toml:13]: Are feature flags really needed here? I think we're always using all of those for every affected crate. Even if not we're using all of those for the whole workspace, so crate will always be compiled with every available feature and not required functions will be removed later on.

**Review by Kononnable — APPROVED** [review]

File: crates/rpc/src/materializer_general.rs (hunk 1 above)
Comment (frondeus) [reply crates/rpc/src/materializer_general.rs:16]: I extracted it to a separate function so I don't have to `map_err` every single time.
TBH, I thought about making these `connect()` in `cargo xtask codegen` but I don't have enough time :)

File: crates/tracing_tools/Cargo.toml (hunk 2 above)
Comment (frondeus) [reply crates/tracing_tools/Cargo.toml:13]: Yeah, for now it makes little sense, but later i'd love to split utils into smaller crates so `data-router` wouldn't use `grpc` feature at all.

**Pull request merged**